src/genie/libs/parser/iosxe/tests/ShowIpSlaSummary/cli/equal/golden_output_1_expected.py

```diff
@@ -0,0 +1,39 @@
+expected_output = {
+    'probes': {
```

Comment:
Let's change to `id` as per the output.
```suggestion
    'id': {
```